app/presenters/prisoner_details_presenter.rb

```diff
@@ -0,0 +1,42 @@
+class PrisonerDetailsPresenter
+  VALID    = 'valid'.freeze
+  INVALID  = 'invalid'.freeze
+  NOT_LIVE = 'not_live'.freeze
+
+  def initialize(prisoner_validation, prisoner_location)
+    self.prisoner_validation = prisoner_validation
+    self.prisoner_location   = prisoner_location
+  end
+
+  def prisoner_existance_status
+    return NOT_LIVE unless Nomis::Api.enabled?
+    case prisoner_existance_error
+    when nil
+      VALID
+    when PrisonerValidation::UNKNOWN, PrisonerLocation::INVALID, PrisonerLocation::UNKNOWN
+      prisoner_existance_error
+    else
+      INVALID
+    end
+  end
+
+  def prisoner_details_incorrect?
```

Comment:
How about `#incorrect_details?` instead? It doesn't read quite right where is used `prisoner_details.prisoner_details_incorrect?` to me. Similar to the rest of the methods.